formiodata/form.py

```diff
@@ -44,9 +47,11 @@ def set_builder_by_builder_schema_json(self):
     def load_components(self):
         for key, component in self.builder.form_components.items():
             # Rather lazy check, but sane.
-            if not self.form.get(key):
-                continue
-            component.value = self.form.get(key)
+            # if not self.form.get(key):
+            #     continue
+            # replaced with default value and fast
+            component.value = self.form.get(key, component.defaultValue)
+            component.render()
```

Comment:
Related to my comment regarding the `html` property in the `Components` class ...
Why setting (rendering) the `html` property here, in the `load_components` function?
I would be careful with potential overhead here and maybe better call the `render` (or `html`) at need.

Reply:
same as above

Reply:
Please could we elaborate on my question here?
In fact I think this won't yet be a problem... But in case there're some heavy `render` implementations implemented, this could cause a performance penalty.

Would it be possible to call/replace the `render` function from elsewhere, at need?
Just implement a `render_components` function which actually does this...

``` python
def render_components(self):
    for key, component in self.components:
        # if not component._is_rendered:
        component.render()
```

And also flag whether it already has been rendered - So re-rendering won't be necessary and avoids a penalty.
Thanks!